### DISCUSSION TITLE
Slider component - question

### DISCUSSION THREAD

**Discussion by adcore-dev** [primefaces/community#687] [PrimeNG] [answered]
Hi everyone,
The question regarding https://primeng.org/slider
I'm seeking a method to display a tooltip showing the current value when clicking on a slider handle, and I want the tooltip to continuously update and remain visible while the handle is being dragged, until it's released. Tooltip should move together with handle. Is this functionality achievable with the current component?

I need something like this:
[https://material.angularjs.org/latest/demo/slider](https://material.angularjs.org/latest/demo/slider)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, such feature is not available yet. Slider doesn't have tooltip support. We'll add it to our internal backlog.
